Umbraco.ModelsBuilder/Umbraco/HashCodeCombiner.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Globalization;
+
+namespace Umbraco.ModelsBuilder.Umbraco
+{
+    // because, of course, it's internal in Umbraco
+    // see also System.Web.Util.HashCodeCombiner
+    class HashCodeCombiner
```

Comment:
NOTE: I've made MB able to see internals in Umbraco .... maybe not Umb core though, not sure.